testing/sdk_cmd.py

```diff
@@ -34,3 +35,12 @@ def run_cli(cmd, print_output=True, return_stderr_in_stdout=False):
     if return_stderr_in_stdout:
         stdout = stdout + "\n" + stderr
     return stdout
+
+
+def convert_output_to_list(output):
```

Comment:
Now that I have a clearer picture of what this does, I think the name is a bit misleading, as it REQUIRES that the input already be the string representation of a python list. Not a blocker though.